packages/block-editor/src/hooks/layout.js

```diff
@@ -95,19 +111,21 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 
 function LayoutTypeSwitcher( { type, onChange } ) {
 	return (
-		<ButtonGroup>
-			{ getLayoutTypes().map( ( { name, label } ) => {
-				return (
-					<Button
+		<BaseControl>
```

Comment:
Why this you add BaseControl here, if anything it feels it should be inside `SegmentedControl` itself no?

Reply:
I remember Q's comment here: https://github.com/WordPress/gutenberg/pull/25674#issuecomment-700122568 and I've following that since 😄

Reply:
All *Control components in `@wordpress/components` package have a `BaseControl` built-in. Should we do the same for `SegmentedControl` ?

Reply:
Wasn't aware of that, thanks! I'll see to it! I have some other enhancements for this control as well, since it was introduced just recently.

Reply:
Created a PR for it here: https://github.com/WordPress/gutenberg/pull/34017